### PR TITLE
Initial update from parsing tts beta file

### PR DESCRIPTION
**Pull request by tonywok**

### CODE REVIEW

File: data/en/cards/alpha/cards.yaml [hunk 1]
```diff
@@ -2,265 +2,306 @@
...
   mana_cost: '3'
   power: '1'
   wind_threshold: '1'
 - identifier: aqueduct
   rarity: elite
   card_type: site
+  release: alpha
   artist: jeff_a_menges
+  keywords:
+  - village
```
Comment (reubenjh) [inline data/en/cards/alpha/cards.yaml:154]: I like the addition of the site types as keywords, nice

File: data/en/cards/alpha/cards.yaml [hunk 2]
```diff
@@ -2,265 +2,306 @@
...
   artist: aronjaart
   name: Atlantean Fate
-  rules_box: ! |
-    Conjure at the intersection of two or four locations of which at least one is a non-ordinary site.
-
-    Affected non-Ordinary sites become water sites and give Water threshold, but lose all other abilities. All minions and relics on the affected sites are forcibly submerged. This is also true for minions and relics that enter afterwards.
-  type_line: An Elite Aura of tragic and entombing tides
-  mana_cost: '3'
+  rules_box: ! |-
+    Affected non-Ordinary sites become Water sites, only provide Water threshold and lose all other abilities. All minions and relics on those sites are submerged and can’t surface.
```
Comment (reubenjh) [inline data/en/cards/alpha/cards.yaml:209]: Thoughts on adding the sugmerge keyword to this?

File: data/en/cards/alpha/cards.yaml [hunk 3]
```diff
@@ -271,568 +312,667 @@
...
+    said the spider to the fly.
+    - "The Spider and the Fly" by Mary Howitt
   fire_threshold: '2'
+- identifier: battering_ram
+  rarity: exceptional
+  card_type: relic
+  release: alpha
+  artist: elwira_pawlikowska
+  keywords:
+  - minion
```
Comment (reubenjh) [inline data/en/cards/alpha/cards.yaml:372]: This looks like a mistake

File: data/en/cards/alpha/cards.yaml [hunk 4]
```diff
@@ -271,568 +312,667 @@
...
+  release: alpha
+  artist: michal_nagypál
+  keywords:
+  - genesis
   name: Brobdingnag Bullfrog
   rules_box: ! |-
-    When this Bullfrog attacks an enemy minion for the first time, it instead swallows that minion and keeps it in its belly until it leaves the Realm.
-  type_line: An Exceptional Beast with a giant appetite
-  mana_cost: '5'
+    Genesis → Swallow a nearby enemy minion. It stays trapped and disabled until Brobdingnag Bullfrog leaves the realm.
```
Comment (reubenjh) [inline data/en/cards/alpha/cards.yaml:613]: Is disabled worth being a keyword? There are 3 cards in the set with it. Maybe not worth it

File: data/en/cards/alpha/cards.yaml [hunk 5]
```diff
@@ -271,568 +312,667 @@
...
+    Double, double toil and trouble; Fire burn and cauldron bubble. -William Shakespeare, Song of the Witches
   fire_threshold: '1'
+- identifier: cave_in
+  rarity: exceptional
+  card_type: magic
+  release: alpha
+  artist: michal_nagypál
+  name: Cave-in
+  rules_box: ! |-
+    Burrow all ground minions on a non-Water site or unburrow all minions there.
```
Comment (reubenjh) [inline data/en/cards/alpha/cards.yaml:696]: Needs burrowing keyword

File: data/en/cards/alpha/cards.yaml [hunk 6]
```diff
@@ -271,568 +312,667 @@
...
   artist: alan_pollack
   name: Cerberus in Chains
   rules_box: ! |-
-    This Cerberus is chained to the site where it was summoned and can't move further than adjacent locations.
-
-    Unleash it if the site moves or leaves the Realm.
-  type_line: An Exceptional Demon on a short leash.
-  mana_cost: '4'
-  power: '5'
+    This Cerberus is chained to the site where it was summoned and is thus immovable while there. Its chain can be destroyed as if it were a relic, thus unleashing this Cerberus.
```
Comment (reubenjh) [inline data/en/cards/alpha/cards.yaml:723]: Does this need the immovable keyword - which actually makes me wonder if that's worth even being a keyword - there are only 3 references in the set now including this

File: data/en/cards/alpha/cards.yaml [hunk 7]
```diff
@@ -271,568 +312,667 @@
...
-  type_line: A Unique Relic of hindered ambition
-  mana_cost: '7'
 - identifier: city_of_agartha
   rarity: unique
   card_type: site
+  release: alpha
   artist: adam_burke
   name: City of Agartha
-  keywords:
-  - burrowing
```
Comment (reubenjh) [inline data/en/cards/alpha/cards.yaml:633]: Believe this should stay

File: data/en/cards/alpha/cards.yaml [hunk 8]
```diff
@@ -271,568 +312,667 @@
...
+  release: alpha
   artist: elwira_pawlikowska
   name: Cloud City
-  rules_box: ! |
-    {Wind:2}: Once during your turn, you may move this City to a free adjacent void.
-  type_line: A Unique Site, lofty and ever fleeting
+  rules_box: ! |-
+    (A)(A): Once on your turn, you may shift this City to an adjacent void.
+  type_line: A Unique Site, lofty and ever fleeting.
   wind_threshold: '1'
```
Comment (reubenjh) [inline data/en/cards/alpha/cards.yaml:775]: Only just noticed this now but I guess we should rename wind_threshold to air_threshold

File: data/en/cards/alpha/cards.yaml [hunk 9]
```diff
@@ -843,326 +983,380 @@
...
-  type_line: Your Avatar of transcendant triumph.
+    Whenever you conjure an aura, you may have an aura you control become a minion with power equal to its mana cost this turn.
+  type_line: Your Avatar weaves magic and land together.
   power: '1'
 - identifier: enlarge
   rarity: exceptional
   card_type: magic
+  release: alpha
   artist: alan_pollack
   name: Enlarge
```
Comment (reubenjh) [inline data/en/cards/alpha/cards.yaml:1361]: I think this got changed to Gigantism and no longer exists 🤔

File: data/en/cards/alpha/cards.yaml [hunk 10]
```diff
@@ -1175,925 +1369,1106 @@
...
   earth_threshold: '1'
 - identifier: entangle_terrain
   rarity: ordinary
   card_type: aura
+  release: alpha
   artist: drew_tucker
   name: Entangle Terrain
   rules_box: ! |-
-    Conjure at the intersection of two or four sites of which at least one is an Earth site.
+    Ground minions here are immobile.
```
Comment (reubenjh) [inline data/en/cards/alpha/cards.yaml:1387]: Keyword immobile

File: data/en/cards/alpha/cards.yaml [hunk 11]
```diff
@@ -1175,925 +1369,1106 @@
...
-  artist: michal_nagypal
+  release: alpha
+  artist: michal_nagypál
   name: Flood
   rules_box: ! |-
-    Conjure at the intersection of two or four locations of which at least one is a Water site.
-
-    Affected sites are Water sites and only provide Water threshold but keep other abilities.
-  type_line: An Exceptional Aura of severe overflow
+    Affected sites are flooded. Lasts 3 of your turns.
```
Comment (reubenjh) [inline data/en/cards/alpha/cards.yaml:1640]: Would have the flood keyword if we made one

File: data/en/cards/alpha/cards.yaml [hunk 12]
```diff
@@ -1175,925 +1369,1106 @@
...
   card_type: site
+  release: alpha
   artist: caio_calazans
+  keywords:
+  - genesis
   name: Floodplain
   rules_box: ! |-
-    The site directly in front of this Floodplain is also a Water site.
-  type_line: An Exceptional Site of an overwhelming flow
+    Genesis → Flood a nearby site this turn.
```
Comment (reubenjh) [inline data/en/cards/alpha/cards.yaml:1653]: Would also have the flood keyword

File: data/en/cards/alpha/cards.yaml [hunk 13]
```diff
@@ -1175,925 +1369,1106 @@
...
   rarity: exceptional
   card_type: minion
+  release: alpha
   artist: ossi_hiekkala
   name: Frontier Settlers
   rules_box: ! |-
-    When these settlers move, they may do so to a void, placing the top site from your atlas there. Once settled, they don't want to leave.
-  type_line: Exceptional Mortals of sweat and survival
-  mana_cost: '3'
+    Frontier Settlers may move into an adjacent void. When they do, place the top site of your atlas there. Once settled, they become immobile.
```
Comment (reubenjh) [inline data/en/cards/alpha/cards.yaml:1697]: immobile keyword

**Review by reubenjh — COMMENTED** [review]
Got up to frontier settlers, will try make some more progress when I get a chance. I'm happy to check into your branch and make the changes we agree on by the way - I was going to make them and PR against your branch but figure it's probably easier to discuss them on here

**Comment by reubenjh** [comment]
Also as a general comment, i noticed there are enough of these sorts of traits now that I think it's worth either adding them as keywords or adding a `traits` list. You've already got `quick` and the site types in there as keywords. 

```
  QUICK = 'quick',
  VILLAGE = "village",
  RIVER = "river",
  TOWER = "tower",
  DESERT = "desert",
  MORTAL = "mortal",
  BEAST = "beast",
  MYTHIC = "mythic",
  DRAGON = "dragon",
  SPIRIT = "spirit",
  MONSTER = "monster",
  DEMON = "demon",
  VEHICLE = "vehicle",
  CONSTRUCT = "construct",
  MONUMENT = "monument"
```

I lean towards keywords as well, as opposed to `traits`. That gives us a way to target things like `Evil Presence` with the `Spirit` keyword, as it wouldn't make sense for the card to have the `Spirit` trait as it isn't a Spirit itself

File: data/en/cards/alpha/cards.yaml [hunk 14]
```diff
@@ -2102,1531 +2477,1793 @@
...
   card_type: minion
+  release: alpha
   artist: tony_szczudlo
   name: Outback Strider
   rules_box: ! |-
-    This Strider must be summoned to any site in the leftmost or rightmost column.
-  type_line: An Exceptional Mortal from a distant land
-  mana_cost: '3'
-  power: '4'
+    This Strider moves freely through sites without any enemies.
```
Comment (reubenjh) [inline data/en/cards/alpha/cards.yaml:3012]: Should these have the movement_x keyword

File: data/en/cards/alpha/cards.yaml [hunk 15]
```diff
@@ -2102,1531 +2477,1793 @@
...
+  release: alpha
   artist: adam_burke
   keywords:
   - airborne
   - haste
-  - voidwalk
   name: Peregrine Apparition
   rules_box: ! |-
-    **Airborne**, **Haste**, **Voidwalk**
+    Airborne, Haste, Voidlwalk
```
Comment (reubenjh) [inline data/en/cards/alpha/cards.yaml:3107]: Typo in voidwalk

File: data/en/cards/alpha/cards.yaml [hunk 16]
```diff
@@ -2102,1531 +2477,1793 @@
...
+  mana_cost: '5'
 - identifier: peregrine_apparition
   rarity: exceptional
   card_type: minion
+  release: alpha
   artist: adam_burke
   keywords:
   - airborne
   - haste
-  - voidwalk
```
Comment (reubenjh) [inline data/en/cards/alpha/cards.yaml:2678]: Need this

File: data/en/cards/alpha/cards.yaml [hunk 17]
```diff
@@ -2102,1531 +2477,1793 @@
...
   name: Peregrine Apparition
   rules_box: ! |-
-    **Airborne**, **Haste**, **Voidwalk**
+    Airborne, Haste, Voidlwalk
 
-    This Apparition moves freely in the Realm.
-  type_line: An Exceptional Spirit of trancendent velocity
+    This Apparition moves freely in the realm.
```
Comment (reubenjh) [inline data/en/cards/alpha/cards.yaml:3109]: Should these have the movement_x keyword

File: data/en/cards/alpha/cards.yaml [hunk 18]
```diff
@@ -2102,1531 +2477,1793 @@
...
   fire_threshold: '1'
   wind_threshold: '1'
+- identifier: ruler_of_thul
+  rarity: unique
+  card_type: minion
+  release: alpha
+  artist: frank_frazetta
+  name: Ruler of Thul
+  rules_box: ! |-
+    This Ruler can move as if the top and bottom edges of the realm were connected, and he moves freely within the top and bottom rows.
```
Comment (reubenjh) [inline data/en/cards/alpha/cards.yaml:3723]: Should these have the movement_x keyword

File: data/en/cards/alpha/cards.yaml [hunk 19]
```diff
@@ -843,326 +983,380 @@
...
   artist: jeff_easley
-  name: East West Dragon
-  rules_box: ! |-
-    **Airborne**
-    As long as this Dragon is in the leftmost or rightmost column, it and allied minions can move freely to any site in those columns.
-  type_line: An Exceptional serpentine Dragon emerges
+  keywords:
+  - airborne
+  name: East-West Dragon
+  rules_box: ! "Airborne\n \nMoves freely while moving sideways."
```
Comment (reubenjh) [inline data/en/cards/alpha/cards.yaml:1301]: Should these have the movement_x keyword

File: data/en/cards/alpha/cards.yaml (hunk 3 above)
Comment (reubenjh) [reply data/en/cards/alpha/cards.yaml:372]: Also needs deathrite keyword

Comment (reubenjh) [reply data/en/cards/alpha/cards.yaml:372]: Though i just noticed this is the only card in the set with it surprisingly

File: data/en/cards/alpha/cards.yaml [hunk 20]
```diff
@@ -2,265 +2,306 @@
...
-  flavor_text: ! |
-    _The lion and the unicorn were fighting for the crown,
-    The lion beat the unicorn all around town,
-    Some gave them white bread, some gave them brown,
-    Some gave them plum cake and drummed them out of town.
-    - English Nursery Rhyme_
+  flavor_text: ! |-
+    "The lion and the unicorn were fighting for the crown. The lion beat the unicorn all around town. Some gave them white bread, some gave them brown. Some gave them plum cake and drummed them out of town." - English Nursery Rhyme
   earth_threshold: '1'
+- identifier: avatar_of_air
```
Comment (reubenjh) [inline data/en/cards/alpha/cards.yaml:250]: This is missing initial_life

File: data/en/cards/alpha/cards.yaml [hunk 21]
```diff
@@ -2,265 +2,306 @@
...
+  artist: séverine_pineaux
+  name: Avatar of Earth
   initial_life: '20'
   rules_box: ! |-
-    Tap → Pay or draw a Site.
+    Tap → Play or draw a site.
 
-    You gain +1 attack power for each nearby Earth site.
-  type_line: Your Avatar of dependable strength
+    You and your Earth sites have +1 Armor.
```
[the remark this answers is not in the document]
Comment (tonywok) [reply data/en/cards/alpha/cards.yaml:272]: Ah, my parser wasn’t looking for it! Good catch will re-generate

File: data/en/cards/alpha/cards.yaml [hunk 22]
```diff
@@ -2,265 +2,306 @@
...
   initial_life: '20'
-  rules_box: ! |-
-    Tap → Pay or draw a site.
-
-    You may move to Water sites You play.
-
-    The site You are on is also a Water site.
-  type_line: Your Avatar of charm and trickery
+  rules_box: ! "Tap → Play or draw a site.\n \nWhenever you play a Water site, you
+    may teleport to it and flood a nearby site until you do so again."
```
[the remark this answers is not in the document]
Comment (tonywok) [reply data/en/cards/alpha/cards.yaml:294]: There’s a section in the rule book called “game concepts”.

Wondering if we should have a similar tagging system to keywords for this.

File: data/en/cards/alpha/cards.yaml [hunk 23]
```diff
@@ -1175,925 +1369,1106 @@
...
+  - submerge
   name: Great Old One
   rules_box: ! |-
-    **Submerge**, **Indestructible**
-    This Old One can surface and attack anywhere in the body of water it is in. At the end of your turn, submerge it.
-  type_line: A Unique Monster stirs beneath the surface
+    Indestructible, Submerge
+  type_line: 'A Unique Monster stirs beneath the surface.
+
+'
```
Comment (reubenjh) [inline data/en/cards/alpha/cards.yaml:1868]: Typo new line chars before the closing quote

File: data/en/cards/alpha/cards.yaml (hunk 4 above)
Comment (tonywok) [reply data/en/cards/alpha/cards.yaml:613]: I’ve been basing it off what’s in the keywords section of the rule book. 

See previous game concepts thought.

File: data/en/cards/alpha/cards.yaml (hunk 7 above)
Comment (tonywok) [reply data/en/cards/alpha/cards.yaml:633]: I think I see what’s happened. My parser is looking for borrowing, but it needs to be looking for borrow as well.

File: data/en/cards/alpha/cards.yaml (hunk 8 above)
Comment (tonywok) [reply data/en/cards/alpha/cards.yaml:775]: Yep, I’m handling this at the ui layer for the moment. I’m thinking I’ll clean them all up in one go once the data is updated.

Will that mess you up? I’m flexible.

File: data/en/cards/alpha/cards.yaml (hunk 9 above)
Comment (tonywok) [reply data/en/cards/alpha/cards.yaml:1361]: Ah, so I should explain this as it’s not obvious.

I removed cards that got removed from the beta that had no deck inclusions.

There’s a handful that did have deck inclusions.

In hindsight, I think I’ll just remove them from peoples’ decks or make a separate file for them temporarily so to not have baggage in this file.

File: data/en/cards/alpha/cards.yaml (hunk 10 above)
Comment (tonywok) [reply data/en/cards/alpha/cards.yaml:1387]: Weird, this almost seems like I have a typo in my parser. Will investigate.

File: data/en/cards/alpha/cards.yaml (hunk 15 above)
Comment (tonywok) [reply data/en/cards/alpha/cards.yaml:3107]: Oh nice, that must be a source typo

File: data/en/cards/alpha/cards.yaml (hunk 17 above)
Comment (tonywok) [reply data/en/cards/alpha/cards.yaml:3109]: There’s a section called “moves freely”, so I think it should have that. Fixing.

File: data/en/cards/alpha/cards.yaml (hunk 22 above)
[the remark this answers is not in the document]
Comment (reubenjh) [reply data/en/cards/alpha/cards.yaml:294]: Ah gotcha, that makes sense. I'm just looking at it from a player perspective on what kinds of things you might consider looking for when building for synergies in the deck building process. Might not be a good approach given it's so subjective

File: data/en/cards/alpha/cards.yaml (hunk 20 above)
Comment (tonywok) [reply data/en/cards/alpha/cards.yaml:250]: Ah that reminds me, they didn’t have it in the data tape gave — was planning to go back. Adding a task for this

File: data/en/cards/alpha/cards.yaml (hunk 23 above)
Comment (tonywok) [reply data/en/cards/alpha/cards.yaml:1868]: Must be in the source. Will fix

File: data/en/cards/alpha/cards.yaml (hunk 7 above)
Comment (reubenjh) [reply data/en/cards/alpha/cards.yaml:633]: + burrowed in this case. Lots of different tenses 😆

File: data/en/cards/alpha/cards.yaml (hunk 3 above)
Comment (tonywok) [reply data/en/cards/alpha/cards.yaml:362]: Added deathrite, trying to get some clarity around the minion subtype. It's weird.

File: data/en/cards/alpha/cards.yaml (hunk 12 above)
Comment (tonywok) [reply data/en/cards/alpha/cards.yaml:1632]: I think I'm inclined to address the flood concern separately if that's okay with you

**Pull request merged**